platform/ios/export/export_plugin.cpp

```diff
@@ -2591,7 +2628,7 @@ Error EditorExportPlatformIOS::_export_project_helper(const Ref<EditorExportPres
 
 		print_line("xcodebuild (.ipa):\n" + export_str);
 		if (!export_str.contains("** EXPORT SUCCEEDED **")) {
-			add_message(EXPORT_MESSAGE_ERROR, TTR("Xcode Build"), TTR(".ipa export failed, see editor log for details."));
+			add_message(EXPORT_MESSAGE_ERROR, TTR("Xcode Build"), vformat(TTR(".ipa export failed, see editor log for details. Log: %s"), export_str));
```

Comment:
Sorry, I added it here because when exporting if there was a failure exporting, I could not find the message. Is there another UX panel that shows the "editor log" or are you talking about the console log? Thanks.